include/websock.js

```diff
@@ -321,26 +341,16 @@ function Websock() {
 
         // private methods
         _encode_message: function () {
-            if (this._mode === 'binary') {
-                // Put in a binary arraybuffer
-                return (new Uint8Array(this._sQ)).buffer;
-            } else {
-                // base64 encode
-                return Base64.encode(this._sQ);
-            }
+            // Put in a binary arraybuffer
+            // according to the spec, you can send ArrayBufferViews with the send method
```

Comment:
I have a vague recollection like I tried that and there were a few browsers that supported arraybuffers but didn't support send of Views. Certainly, that was a long time ago so probably irrelevant, but that's why it wasn't already doing that.